test/tests/performance/pbs_sched_perf.py

```diff
@@ -216,3 +216,33 @@ def test_pset_fuzzy_perf(self):
             cycle1_time, cycle2_time, (cycle1_time / cycle2_time) * 100))
         self.assertLess(cycle2_time, cycle1_time,
                         'Optimization was not faster')
+
+    @timeout(3600)
+    def test_sched_query(self):
+        """
+        Test scheduler's turnaround time with focus on querying the universe
+        """
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'False'})
+
+        # We have 10010 vnodes, submit 10050 jobs so that it can run almost
+        # all of them
+        a = {'Resource_List.select': '1:ncpus=1'}
+        jids = self.submit_jobs(attribs=a, num=10050, step=0, wt_start=900)
```

Comment:
I think we should submit jobs with the environment so that the job size becomes big enough that it makes a difference querying limited attributes.